pkg/bff/users.go

```diff
@@ -234,7 +234,7 @@ func (s *Server) Login(c *gin.Context) {
 	}
 
 	// If the user app metadata has changed, set the refresh flag in the response
-	if !oldAppdata.Equals(appdata) {
+	if appdata.Equals(oldAppdata) {
```

Comment:
If `appdata` equals `oldAppdata` doesn't that imply that it hasn't changed? I guess I'm not understanding this change if you wouldn't mind help clarifying -- and if the logic has changed maybe the comment also needs an update?

Reply:
Whoops, I think we got borked by merge conflict resolution here. Thanks for catching it!